resources/python/install_source_only.sh

```diff
@@ -59,6 +60,14 @@ rm -Rf $source_dir/incremental/tests
 # into other processes, which we don't require. That module is compiled, so we'll delete it.
 rm -Rf $source_dir/twisted/python/_sendmsg.so
 
+# Compress all files
+cd $source_dir
+zip -q -r pkgs.zip ./*
+cd $build_dir
```

Comment:
I prefer the pushd/popd approach. Or I also recently learned `cd -`

Reply:
Can you elaborate more?

Reply:
```bash
pushd $source_dir
zip -q -r pkgs.zip ./*
popd
```

Reply:
TIL pushd/popd. Thanks!